src/ir_Hitachi.cpp

```diff
@@ -711,10 +726,8 @@ stdAc::state_t IRHitachiAc424::toCommon(void) {
   result.degrees = this->getTemp();
   result.fanspeed = this->toCommonFanSpeed(this->getFan());
 
-  // TODO(jamsinclair): Add support.
-  result.swingv = stdAc::swingv_t::kOff;
-
   // Not supported.
+  result.swingv = stdAc::swingv_t::kOff;
```

Comment:
While it isn't clear from the rest of the code in IRac.cpp etc. You should use something like:
```
result.swingv = (this->getButton() == kHitachiAc424ButtonVSwing) ? stdAc::swingv_t::kAuto : stdAc::swingv_t::kOff;
```

We can support _toggled_ states in the "Common" a/c stuff with appropriate code. I'll try to add stuff to this PR tomorrow for it.

We do this by comparing the previous state to the new/desired state in the `IRac` object and calculating if we need to send a toggle or not.

Reply:
Done